data/jibauni/jibauni missions.txt

```diff
@@ -0,0 +1,579 @@
+				`	"Alright."`
+				`	"Nope, sorry. I agreed to take you... once. No more."`
+					goto declining?
+			`	"Perfect! I will be ready when you are."`
+				accept
+			label declining?
+			`	"You can not do that! It is not very kind; there is unlikely to be anybody who will take me back to Jibauni space. Please take me!"`
+			choice
+				`	"Oh, whatever. I'll take you to <planet>."`
+					accept
+				`	"My word is my word. Goodbye."`
+			action
+				"reputation: Jibauni" -= 3
+			`	Dran-lin continues to protest, until you leave, slamming the door behind you. The Jibauni may or may not be happy about this.`
+				decline
+	on complete
+		conversation
+			`Dran-lin leaves the <ship> for a scouting run. "I will be back soon," he promises.`
+			branch "dran-lin put in jail"
+				random < 3
+			choice
+				`	(Wait.)`
+			branch "dran-lin likes"
+				random < 60
+				not "jibauni: dran-lin: says rebels are good"
+			branch "dran-lin likes with input"
+				random < 50
+				has "jibauni: dran-lin: says rebels are good"
+			label "dran-lin dislikes"
+			action
+				event "jibauni: dran-lin abandoned" 1
+			`	He is back soon. "I really do not like this place," he announces. "It is even worse than <origin>, and that is saying a lot. Please meet me in the spaceport at the Indon-alo in an hour. I am going to the capital to ask a few questions. I mean, I guess I can stay if you do not come, but it would really be much appreciated if you do."`
+			branch "dran-lin put in jail"
+				random < 15
+			`	You head to the Indon-alo to wait for Dran-lin.`
+				goto end
+			label "dran-lin likes"
+			action
+				payment 5000
+				set "jibauni: dran-lin stays with the rebels"
+			`	He arrives back at your ship soon. "You know, Captain, I think I like this place. Thanks for bringing me! Here is a payment of <payment> for that." He disappears into the spaceport again.`
+				goto end
+			label "dran-lin likes with input"
+			action
+				payment 5000
+				set "jibauni: dran-lin stays with the rebels"
+			`	Dran-lin is soon back. "<last>, thanks for bringing me here! It is a really great place - you were right when you said the Rebel Jibauni were good. Or should I say Real Jibauni?" He trails off, and soon leaves, after giving you <payment> in chips of a credit each.`
+				goto end
+			label "dran-lin put in jail"
+			`	He doesn't come. After hours of waiting, you decide that you will go and try to find Dran-lin. After a few hours of asking around, government officials tell you that he's been put in jail, though they won't give you a reason. "It will cost five thousand credits to get him out of there," they say.`
+			choice
+				`	"I'll pay that."`
+				`	"Of course not! That's outrageous! He can stay there for all I care."`
+					goto dranlinimprisoned
+			action
+				payment -5000
+			`	Dran-lin is released immediately. "Thank you," he says when he learns who paid for the release. "I have one more thing to wrap up, but then meet me in the spaceport. I am going home."`
```

Comment:
```suggestion
			`	Dran-lin is released immediately. "Thank you," he says when he sees you, obvious relief flooding his alien face. "I have one more thing to wrap up, but then meet me in the spaceport. I am going home."`
```

Could there be an option here for they ask to leave immediately, and you do a launch command?